api/samples/cbr.c

```diff
@@ -232,10 +232,9 @@ insert(hash_table_t table, app_pc addr, cbr_state_t state)
 static void
 at_taken(app_pc src, app_pc targ)
 {
-    dr_mcontext_t mcontext = {
-        sizeof(mcontext),
-        DR_MC_ALL,
-    };
+    dr_mcontext_t mcontext;
```

Comment:
These are good cleanups: I didn't realize there were so many like this in the samples and tests.  But given that there are quite of few of these mcontext changes I would separate these into their own PR (no need for an issue: iX branch) since they are logically distinct.  Cleaner history, simpler revert paths, etc.

Reply:
Makes sense. Sent PR #4499 for review.